File: app/src/main/java/com/arjun/deeper/views/home/InterfaceHome.java
package com.arjun.deeper.views.home;

import com.arjun.deeper.baseclasses.IBasePresenter;
import com.arjun.deeper.baseclasses.IBaseView;

public interface InterfaceHome {

    interface IActivity extends IBaseView {

        void hideStatusBar();

        void hideActionBar();

        void showActionBar();
    }

    interface IPresenter extends IBasePresenter<IActivity> {

    }
}

File: app/src/main/java/com/arjun/webapp/views/home/InterfaceHome.kt
package com.arjun.webapp.views.home

import com.arjun.webapp.baseclasses.IBasePresenter
import com.arjun.webapp.baseclasses.IBaseView

interface InterfaceHome {
    interface IActivity : IBaseView {
        fun hideStatusBar()
        fun loadUrl(url: String)
    }

    interface IPresenter : IBasePresenter<IActivity>
}